### ISSUE TITLE
bower.json missing angular-ui?

### ISSUE THREAD

**Issue by tarlepp** [UniBot/UniBot#43]
Seems like many of the plugins uses angular UI ```pagination``` which required angular-ui.

**Comment by ProLoser** [comment]
Yes, I didn't bother porting pagination over during the refactor as it
required some coordination with the angular controller. Since I was trying
to 'agnosticize' the plugins from the controller logic, I put the
pagination stuff on pause. I suppose we can just have some sort of open
convention for hooking into pagination in the controller if you want to,
but it might be better to somehow allow the plugin developer to supply an
angularjs controller too (which makes things a little bit trickier).
